src/containers/Application/ApplicationList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Container, Table, List, Label } from 'semantic-ui-react'
+import { Loading, FilterList } from '../../components'
+import { useRouter } from '../../utils/hooks/useRouter'
+import useListApplication from '../../utils/hooks/useListApplications'
+
+const ApplicationList: React.FC = () => {
+  const { query } = useRouter()
+  const { error, loading, applications } = useListApplication()
+
+  return error ? (
+    <Label content="Problem to load applications list" error={error} />
+  ) : loading ? (
+    <Loading />
+  ) : (
+    <Container>
+      <FilterList />
+      {Object.keys(query).length > 0 && <h3>Query parameters:</h3>}
+      <List>
+        {Object.entries(query).map(([key, value]) => (
+          <List.Item key={`app_selected_parameter_${value}`} content={key + ' : ' + value} />
+        ))}
+      </List>
+      {applications && applications.length > 0 && (
+        // TODO: Implement sortable columns -> Requires reducer
+        <Table sortable stackable selectable>
+          <Table.Header>
+            <Table.Row>
+              {Object.entries(applications[0]).map(([key]) => (
+                <Table.HeaderCell key={`app_header_${key}`}>{key}</Table.HeaderCell>
+              ))}
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            {applications.map((application, index) => (
+              <Table.Row key={application.serial}>
+                {Object.values(application).map((value) => (
+                  <Table.Cell key={`app_${index}_${value}`}>{value}</Table.Cell>
```

Comment:
Small thing: maybe the key should be `app_${index}_${Object.keys(application)[index]}` so it'll be `app_0_status` rather than `app_0_DRAFT` -- seems a bit more descriptive and it'll solve the above problem of having `null` values.

I know it's a bit ugly having an additional `Object.keys`, but I'm about to use it here in #67 anyway, so why not assign the current key to a variable at the start of the per-cell loop, and then we can use it freely wherever within the row?

Reply:
Ignore this suggestion, [this one](https://github.com/openmsupply/application-manager-web-app/pull/107#discussion_r520173067) is better, sorry.

Reply:
Yes. Indeed. Thanks for this suggestion!

Reply:
Well, I think I'll be combining with your next suggestion to make it not have to do this Object.keys inside the call